sourmash/signature_json.py

```diff
@@ -253,32 +245,62 @@ def save_signatures_json(siglist, fp=None, indent=None, sort_keys=True):
         x.append(sketch)
         top_records[k] = x
 
-    if not top_records:
-        return ""
+    return top_records
 
+
+def add_meta_save(top_records):
+    """ Convert one signatures into a JSON dict
+    - siglist: sequence of SourmashSignature objects
+    - index: index of siglist to save
+    """
+    notify("in add_meta_save", end="\r")
+    from .signature import SIGNATURE_VERSION
     records = []
     for (name, filename), sketches in top_records.items():
         record = {}
         if name:
-            record['name'] = name
+            record["name"] = name
```

Comment:
please don't change things unnecessarily as it complicates the diffs - can you revert this back to single quotes, pls?

Reply:
still double quotes? :)

Reply:
changed now, missed it, sorry!